resources/views/dash.blade.php

```diff
@@ -20,74 +23,128 @@
       </div>
     @endif
 
-  <p>{{$user->getDisplayName()}}</p>
-
-  <!--If the user logged in is the same user that owns the profile, display profile customization features -->
-  @if(Auth::user()->id == $user->id)
-    
-    <div>
-      <span>Update Username</span>
-      <form enctype="multipart/form-data" action="{{route('dash.update_profile', ['id' => $user->getDisplayName()])}}" method="POST">
-        @csrf
-        <label>Change Photo</label>
-        <input type="file" name="avatar"></input>
-        <br>
-        <label>Change Name</label>
-        <input type="text" name="name" value="{{$user->getName()}}"></input>
-        <br>
-        <label>Change Username</label>
-        <input type="text" name="new_display_name" value="{{$user->getDisplayName()}}"></input>
-        <span>Change Pronouns</span>
-        <input type="text" name="pronouns" value="{{$user->getPronouns()}}"></input>
-        <span>Change Bio</span>
-        <input type="text" name="bio" value="{{$user->getBio()}}"></input>
-        <br>
-        <span>Change Age</span>
-        <input type="text" name="age" value="{{$user->getAge()}}"></input>
+  <style type="text/css">
+    .content-section {
+      display: none;
+    }
+
+    .boldText {
+    font-weight:bold;
```

Comment:
Not sure where this is used, but bootstrap has a "font-weight-bold" class that we can use if we want to bold the font of an element. We should use that instead.